catalog/ui/search-ui-app/src/main/resources/etc/org.codice.ddf.catalog.ui.config.config

```diff
@@ -45,12 +45,6 @@ scheduleFrequencyList=L[ \
   "57600", \
   "86400", \
   ]
-hiddenAttributes=[ \
```

Comment:
❓ Since you're already modifying the config quite a bit, would be be okay to go ahead and change the pid (and the name of the file) so that it doesn't end in `.config.config`? We can add a note in the release notes about the name change.

Reply:
What would you like to call it?

Reply:
`base.config`?

Reply:
Probably just `org.codice.ddf.catalog.ui.config`

Reply:
How about just `org.codice.ddf.catalog.ui`? So then the config files would be `org.codice.ddf.catalog.ui.config` and `org.codice.ddf.catalog.ui.theme.config`.

Reply:
So I can't call it that because that file already exists. Am I good to delete the existing `org.codice.ddf.catalog.ui.config` in https://github.com/codice/ddf/tree/master/catalog/ui/search-ui-app/src/main/resources/etc?

Reply:
Yes. I think you can go ahead and remove/rename what makes sense to you.